demux/utils/novaseq_samplesheet.py

```diff
@@ -185,26 +207,32 @@ def pad_and_rc_indexes(
 
         return index1, index2
 
-    def get_reagent_kit_version(self) -> int:
+    def get_reagent_kit_version(self) -> float:
         """ Derives the reagent kit version from the run parameters """
 
         parameter_to_version = {"1": 1.0, "3": 1.5}
+        reagent_kit_version = self.runparameters.reagent_kit_version
+        if reagent_kit_version not in parameter_to_version.keys():
+            raise NoValidReagentKitFound(
+                f"Expected reagent kit version 1 or 3, found {reagent_kit_version} instead. Exiting"
```

Comment:
Maybe this should iterate the dictionary of parameters to version so that if we alter it in the future we don't have to update this text?

Reply:
fixed in https://github.com/Clinical-Genomics/demultiplexing/pull/146/commits/e61c4a043c4e1be96d64b0f18e2a15725abf2056